src/components/HomePage/ContactUs.vue

```diff
@@ -104,7 +104,8 @@ export default {
       phone_number: "",
       preferred_method_of_communication: "",
       message: "",
-      form_sent: false
+      form_sent: false,
+      formURL: process.env.NODE_ENV == "development" ? process.env.VUE_APP_URL_CONTACT_STAGING : process.env.VUE_APP_URL_CONTACT_PRODUCTION
```

Comment:
we don't need this check, just set the one variable, don't set either for staging or prod

Reply:
ok